frontend/src/views/welcome/Login.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <AppRow>
+    <AppCol class="col3">
+      <section id="menu">
+        <ul>
+          <li><a href="#">What is new MSP Direct?</a></li>
+          <li><router-link :to="{ name: 'CredentialsInfo' }">What are new log-in Credentials</router-link></li>
+          <li><a href="https://www2.gov.bc.ca/gov/content/health/practitioner-professional-resources/system-access/msp-direct" target="_blank">Transactions in MSP Direct?</a></li>
```

Comment:
Should we include an icon to indicate the link is opening in a new tab?

Reply:
Yeah, a little plus/arrow/popout icon would be good. I'll see what's available to us.

Reply:
Updated.